.eslintrc.js

```diff
@@ -34,6 +33,7 @@ module.exports = {
     'key-spacing': 'warn',
     'keyword-spacing': 'warn',
     'lines-between-class-members': 'warn',
+    '@typescript-eslint/member-delimiter-style': ['warn', { 'multiline': { 'delimiter': 'none' } }],
```

Comment:
I'm in favor of this change, but since this requires changing existing code (DataTableProvider.ts) and ESLint changes affect all code, I think it is worth its own PR. @kyuksel since you've been working on coding guidelines, what do you think?

Reply:
P.S. I ❤️ the linting rule. Thanks for thinking of it @jdcanas 😊

Reply:
agreed.  This linter rule change also adds more noise to this PR.

Reply:
Moved to this PR https://github.com/DataBiosphere/terra-ui/pull/3496. Easiest way to do this was to squash, soft reset, then re-commit the changes that did not have the lint rule noise. As such, PR chat history might be wonky, but I found this trade-off to be better than relying on me to transfer the changes to another branch rather than git.